README.md

```diff
@@ -51,3 +57,15 @@ RsyncLog           | Open log file for rsync.nvim.
 RsyncProjectConfig | Print out current project config.
 
 *: Files which are excluded are, everything in .gitignore and .nvim folder.
+
+## Configuration
```

Comment:
I think this part is unnecessary you are just duplicating the values from the installation step